packages/valory/skills/decision_maker_abci/policy.py

```diff
@@ -21,12 +21,18 @@
 
 import json
 import random
-from dataclasses import asdict, dataclass, is_dataclass
-from typing import Any, List, Optional, Union
+from dataclasses import asdict, dataclass, field, is_dataclass
+from typing import Any, Dict, List, Optional, Union
+
+from packages.valory.skills.decision_maker_abci.utils.scaling import scale_value
 
 
 RandomnessType = Union[int, float, str, bytes, bytearray, None]
 
+VOLUME_FACTOR_REGULARIZATION = 0.5
+UNSCALED_WEIGHTED_ACCURACY_INTERVAL = (0.5, 100.5)
```

Comment:
For the release I prefer that we already correct the range to be (-0.5, 100.5)

Reply:
10231884d190f672ff2c726ff5e0fa27c5390ceb